index.html

```diff
@@ -338,42 +295,25 @@ <h1>Prototype Viewer MediaWiki 3D GLTF Data, STL data is already available </h1>
             // this.progressBar.animateTo(5);
             this.promise.then((object) => {
                 delete this.promise;
-                // check for GLB
-                if (this.modeltype === 'glb') {
-
-                    object.scene.traverse(function (node) {
-                        console.log(node);
-                        if (node.material) {
-                            node.material.side = THREE.DoubleSide;
-                        }
```

Comment:
I don't recommend overriding glTF materials with `DoubleSide`. Unlike STL, a glTF model can be authored to specify front-sided vs. double-sided internally. By overriding it we may break the author's intended rendering. Also, double-sided rendering is more expensive.

Reply:
OK, I understand. I use the WIDAR software for my models (Ganesha, Lion). Unfortunately, DoubeSide does not seem to be taken into account there. But I can certainly transform this too.